docs/building_the_project.rst

```diff
@@ -534,6 +562,48 @@ definitions in 2 ways:
   `DPC++ User Manual <https://intel.github.io/llvm-docs/UsersManual.html>`_
   for more information on ``-fsycl-targets``.
 
+Building for portFFT
+^^^^^^^^^^^^^^^^^^^^^^
+
+Note the portFFT backend is experimental and currently only supports a
+subset of the operations and features.
+The portFFT backend uses the `portFFT <https://github.com/codeplaysoftware/portFFT>`_
+project as a header-only library.
+
+* On Linux*
+
+.. code-block:: bash
+
+   # Inside <path to onemkl>
+   mkdir build && cd build
+   cmake .. -DENABLE_PORTFFT_BACKEND=ON \
+            -DENABLE_MKLCPU_BACKEND=OFF  \
+            -DENABLE_MKLGPU_BACKEND=OFF  \
+            -DTARGET_DOMAINS=dft \
+            [-DPORTFFT_REGISTERS_PER_WI=128] \ # Example portFFT tuning parameter
+            [-DREF_BLAS_ROOT=<reference_blas_install_prefix>] \ # required only for testing
+            [-DPORTFFT_DIR=<path to portBLAS install directory>]
+   cmake --build .
+   ./bin/test_main_blas_ct
```

Comment:
```suggestion
   ./bin/test_main_dft_ct
```

Reply:
maybe check that that is the name

Reply:
That is indeed the correct name. Fixed in [55b1119](https://github.com/oneapi-src/oneMKL/pull/457/commits/55b11197a422e252954e5a423249423e77dbdfc1).